.circleci/config.yml

```
@@ -0,0 +1,46 @@
+version: 2.1
+
+orbs:
+  codacy: codacy/base@5.0.5
+  codacy_plugins_test: codacy/plugins-test@0.14.6
+
+workflows:
+  version: 2
+  compile_test_deploy:
+    jobs:
+      - codacy/checkout_and_version:
+          write_sbt_version: true
+      - codacy/sbt:
+          name: publish_docker_local
+          cmd: |
+            sbt "set scalafmtUseIvy in ThisBuild := false;
+                 scalafmt::test;
+                 sbt:scalafmt::test;
+                 set name := \"$CIRCLE_PROJECT_REPONAME\";
+                 universal:stage"
+            docker build -t $CIRCLE_PROJECT_REPONAME:latest .
+            docker save --output ~/workdir/docker-image.tar $CIRCLE_PROJECT_REPONAME:latest
```

You can omit workdir here:
```suggestion
            docker save --output docker-image.tar $CIRCLE_PROJECT_REPONAME:latest
```